.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
 python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
-
-install: pip install pipenv tox-travis  # The test command will install all necessary other deps
+- '2.7'
+- '3.5'
+- '3.6'
+install: pip install pipenv tox-travis
 script: make ci-test
+
+deploy:
+  provider: pypi
+  user: dfuchs
+  password:
+    secure: CZcHt9q4RND+Ttt0CxBtArBCd8X88MmyzNSVc+bH3tH0XaH2muF4NOH2VX7zH7ESUvHjj/PzBnZ29We3tKnKPbV9fZHI/fnxKAwi+8GlfnNdl3KiCgjvzgkFiz1u5PwYJHSRplBnd8mlPSo9JWNKxXKDiftppqd2nURlnSntHBoYfdhUoX5HOoGdd/pSu4M701GFCtEbeuqcotlk/sRKZFxb7m5cPPph9QfXm+Le4W13a8eCz3uRGC304x/+QkHunay3boLDBSZwZA0vRq7NzhXWo59IkAwJIMAgKlEs/Qjz2ny8hhpxi/2NBh920+vFOT22mkb74rMHES/iNDx1XS6m+VX5jzLICVeZQof5U3vm/OCXq7HySbmv+NvX6jv5FexiMFuyWv4ysnzcWSEcI90nRSjsQWfH3pTdeYlLnpC/BwnMYjDNSXUzHjtRDAn9hbusnVIMlAMWcxquQP4y31GfIFOvQxeTDU5TMqZ2Ywpae6oGALjbIxtJypznWeODIPzpZ5eLHtleBhnjwOG3ASu0gx0awgRl+BlzV0claUJtfXBT9TkICOeDntV4P00V1yTCxuWHNVV9IckzvCcP/rFCDgxR5I5upGHqWHhRAvllrNEckhXiBH1EtSi12EB9blhxfeP2D+UBxFY3/gMAXXHpJC9rx2oLnKpvSuqQ7yM=
```

Comment:
Hi @fajpunk , did you accidentally just release your pypi password publicly?
(Sorry, I've been looking for ways to reach you privately, but could not find any - so hence a public message).
Cheers,
Vlad

Reply:
Hi @vladimir-mencl-eresearch, thanks for the heads up.  That string is my PyPI password encrypted using the TravisCI client's `encrypt` command, which, as I understands it, encrypts it using a private key associated with this repository, and only readable by the TravisCI system:

https://docs.travis-ci.com/user/encryption-keys/

If I correctly understand the way this works, Travis generates an RSA keypair for this repository, and encrypts that value using the public key from that keypair.  The Travis system that runs the CI jobs have access to the private key, so it can be decrypted when the job is run (and it doesn't show up in the output).  So, there's no security risk by checking this encrypted value into public version control, given:
  * No one can reverse the encryption
  * I trust the Travis folks not to have access to or use the private key

To be honest, it still creeps me out a little and I did change the password after I pushed this commit ;)  But theoretically, it should be safe to have this out there in public github.